src/main/java/core/basesyntax/Application.java

```diff
@@ -3,11 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         Lottery lottery = new Lottery();
-        Ball ball1 = lottery.getRandomBall();
-        Ball ball2 = lottery.getRandomBall();
-        Ball ball3 = lottery.getRandomBall();
-        ball1.toString();
-        ball2.toString();
-        ball3.toString();
+        for (int i = 0; i < 3; i++) {
+            Ball ball = lottery.getRandomBall();
+            System.out.printf(ball.toString());
```

Comment:
it's unnecessary to call toString() method on ball because System.out.println does it automatically